mne/time_frequency/spectrum.py

```diff
@@ -1280,7 +1367,10 @@ class EpochsSpectrum(BaseSpectrum, GetEpochsMixin):
         have been computed.
     %(info_not_none)s
     method : str
-        The method used to compute the spectrum ('welch' or 'multitaper').
+        The method used to compute the spectrum (``'welch'`` or ``'multitaper'``).
```

Comment:
Let's make @hoechenberger happy and annotate this as `Literal["welch", "multitaper"]` instead of just `str`. We can then drop the parenthetical from the description too.

Reply:
https://github.com/mne-tools/mne-python/blob/322ae92f4879d24a31d9ddbd1c504c594dbd1966/mne/time_frequency/spectrum.py#L1092-L1093

And did same for `EpochsSpectrum`